lib/radiator_web/router.ex

```diff
@@ -38,6 +38,9 @@ defmodule RadiatorWeb.Router do
 
     get "/feed/:podcast_id", FeedController, :show
 
+    get "/audio/:audio_id/player.json", PlayerController, :audio_config
+    get "/episode/:episode_id/player.json", PlayerController, :episode_config
```

Comment:
imho those routes should not exist this way. we need a public route infrastructure, and these need to be under whatever the podcast/episode route is. But probably okay for now, maybe add a comment? I would also like us very much to add some indirection to the id for public urls. E.g. Either have a radical uuid to internal id mapping, or something more semantic and nice. But exposing the internal database id's for the public pages I'm not happy with.